contrib/hooks/add_delete_configopt.py

```diff
@@ -0,0 +1,30 @@
+# Small example of how to add/delete a configure option.
+
+# We need to be able to distinguish between versions of OpenMPI
```

Comment:
should mention author here, so people know who to talk to if they have questions or remarks?